### PR TITLE
feat: timer and questions

### PR DESCRIPTION
**Pull request by AlefrankM**

### CODE REVIEW

**Review by angelstackit — APPROVED** [review]

File: src/interviews/gateways/timer.gateway.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  cors: { origin: '*' },
+  transports: ['websocket']
+})
+export class TimerGateway implements OnGatewayInit, OnGatewayConnection, OnGatewayDisconnect {
+  @WebSocketServer() server: Server
+
+  constructor(private readonly interviewRepository: InterviewsRepository) {}
+
+  afterInit() {
+    console.log('Socket started')
```
Comment (juliswer) [inline src/interviews/gateways/timer.gateway.ts:19]: remove clg

File: src/interviews/gateways/timer.gateway.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  }
+
+  handleConnection() {
+    console.log('Hi 👌👌👌')
```
Comment (juliswer) [inline src/interviews/gateways/timer.gateway.ts:23]: remove clg

File: src/interviews/gateways/timer.gateway.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  }
+
+  handleDisconnect() {
+    console.log('Bye')
```
Comment (juliswer) [inline src/interviews/gateways/timer.gateway.ts:27]: remove clg

**Review by juliswer — APPROVED** [review]
LGTM. Please remove clgs before merging.
Amazing work!!!!

**Pull request merged**